givitsite/coordinate/models.py

```diff
@@ -1,3 +1,42 @@
 from django.db import models
+from django import forms
+from friendreq.models import REGION_CHOICES, ITEM_CHOICES , ItemRequest 
 
-# Create your models here.
+
+
+class CoordinatedItems(models.Model):    
+    item = models.CharField(max_length = 40,choices = ITEM_CHOICES)
+    request_id = models.IntegerField(default = 100)
+    date = models.DateField(auto_now=False, auto_now_add=False)
```

Comment:
a field named date can cause problems as that is often a reserved word. you might want it to be created_date, updated_date, or similar